src/compiler/checker.ts

```diff
@@ -33486,9 +33486,9 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
             (typeArguments.length >= minTypeArgumentCount && typeArguments.length <= numTypeParameters);
     }
 
-    // If type has a single call signature and no other members, return that signature. Otherwise, return undefined.
-    function getSingleCallSignature(type: Type): Signature | undefined {
-        return getSingleSignature(type, SignatureKind.Call, /*allowMembers*/ false);
+    // By default if type has a single call signature and no other members, return that signature. Otherwise, return undefined.
+    function getSingleCallSignature(type: Type, allowMembers = false): Signature | undefined {

```

Comment:
same question here, although it likely has way more calls than the other two functions